bdd/spec/016_type_spec.sh

```diff
@@ -61,4 +61,47 @@ Describe "Type detection"
       The output should eq "$GENTYPEOUTPUT"
     End
   End
+
+  Describe "using non-imported type returned by imported function"
+    before() {
+      sourceToTemp "
+        from @std/app import start, exit
+        from @std/http import fetch, Request
+
+        on start {
+          arghFn('{\"test\":\"test\"}');
+          emit exit 0;
+        }
+
+        fn arghFn(arghStr: string) {
+          fetch(new Request {
+              method: 'POST',
+              url: 'https://reqbin.com/echo/post/json',
+              headers: newHashMap('Content-Length', arghStr.length().toString()),
+              body: arghStr,
+            });
+        }
+      "
+    }
+    BeforeAll before
+
+    after() {
+      cleanTemp
+    }
+    AfterAll after
+
+    OUTPUT="{\"success\":\"true\"}"
```

Comment:
This test doesn't actually output anything because it ignores the `Result<Response>`

Reply:
ah right